build.gradle

```diff
@@ -44,6 +44,10 @@ subprojects {
         )
       }
     }
+
+    dependencies {
+      testCompile('junit:junit:4.12')
+    }
```

Comment:
Is it needed for this pull request?

In the future, what do you think of keeping different changes in separate commits?